packages/bpk-component-loading-button/src/BpkLoadingButton.js

```diff
@@ -113,36 +113,31 @@ const BpkLoadingButton = (props: LoadingProps) => {
   const loadingIcon = getLoadingIcon(props);
 
   const classNames = getClassName(
-    loading && 'bpk-loading-button',
-    loading && (link || linkOnDark) && 'bpk-loading-button--link',
     className,
   );
 
   const iconClassNames = getClassName(
     'bpk-loading-button__icon',
-    large && 'bpk-loading-button__icon--large',
-    iconOnly &&
-      (large
-        ? 'bpk-loading-button__icon--large-icon-only'
-        : 'bpk-loading-button__icon--icon-only'),
     (link || linkOnDark) && 'bpk-loading-button__icon--link',
   );
 
   return (
     // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
     <BpkButton
```

Comment:
We should also possibly use this as an opportunity to migrate this component to Buttonv2?